package.json

```diff
@@ -21,7 +21,7 @@
     "doctoc": "doctoc --title '## Table of Contents'",
     "format": "prettier --cache --write '**/*.{js,md,scss,ts,tsx}' '!**/{build,es,lib,storybook,ts,umd}/**'",
     "format:diff": "prettier --list-different '**/*.{js,md,scss,ts,tsx}' '!**/{build,es,lib,storybook,ts,umd}/**' '!packages/components/**'",
-    "lint": "eslint actions config packages www",
+    "lint": "eslint --cache --fix --max-warnings 0 actions config packages www",
```

Comment:
I don't think `--cache` and `--fix` should be on by default so `yarn lint` remains deterministic in CI. Both can/should be turned on within a dev's IDE though. Maybe it would make sense for the precommit hook could use them. A new `lint:fix` script with both these options could be nice too.

Will `--max-warnings 0` cause it to always fail, even when there are 0 warnings? I assume the default `-1` means it's turned off and always passes, but not sure. Would we want `--max-warnings 1`?